ail_vru_uc1_scenario/SubsystemControllerParams.yaml

```diff
@@ -23,8 +23,6 @@ hardware_interface:
         - /hardware_interface/carla_gnss_driver
       ros1_required_drivers:
         - /hardware_interface/carla_driver
-      ros1_camera_drivers:
-        - /hardware_interface/carla_camera_driver
```

Comment:
Just a note - this PR has overlapping changes from https://github.com/usdot-fhwa-stol/carma-config/pull/387/files